packages/app-builder-backend/src/utils/graphqlFetch.ts

```diff
@@ -0,0 +1,41 @@
+import { DocumentNode } from 'graphql'
+import fetch from 'node-fetch'
+import { graphqlUri } from '../config.json'
+
+type AuthTokens = {
+  accessToken: string
+  idToken: string
+}
+
+const getGqlString = (doc: DocumentNode) => doc.loc && doc.loc.source.body
+
+export const query = async <T>(
+  query: DocumentNode,
+  variables: Record<string, any>,
+  dataKey: string,
+  { idToken, accessToken }: AuthTokens,
+) => {
+  const res = await fetch(graphqlUri || 'https://graphql.dev.paas.reapit.cloud/graphql', {
```

Comment:
Would be good if this came from the config.json

Reply:
it does, i'm just providing a default for types sake